wiki/Game_modifier/Hidden/id.md

```diff
@@ -0,0 +1,81 @@
+---
+stub: true
+tags:
+  - hidden
+  - HD
+  - mod
+  - game modifier
+---
+
+# Hidden (mod)
+
+![Tampilan permainan osu! dengan menggunakan mod HD](/wiki/shared/mods/HD.png "Ikon mod Hidden (HD)")
```

Comment:
```suggestion
![Ikon mod HD](/wiki/shared/mods/HD.png "Ikon mod Hidden (HD)")
```

Reply:
that was a weeeeeeiiiiiirrdd one, thanks